tsconfig.json

```diff
@@ -6,6 +6,6 @@
     "sourceMap": true,                     /* Generates corresponding '.map' file. */
     "alwaysStrict": true,                  /* Parse in strict mode and emit "use strict" for each source file. */
     "baseUrl": "./Samples-Typescript",
-    "typeRoots": ["./node_modules/@tableau", "./node_modules/@types"]
+    "types": ["@tableau/extensions-api-types", "jquery"]
```

Comment:
can this be `@tableau/extension-api-types` and `./node_modules/@types`?

Reply:
Tried that, unfortunately does not work. typeRoots can't be pointed at projects, and types overrides typeroots entries.

Reply:
Also, it seemed like specifically calling out your types usage was the 'best practice' when I was bouncing around on docs pages for this.

Reply:
Also also, I was looking for this:
https://github.com/microsoft/TypeScript/issues/18588
Which does not exist as well.